MfciPkg/Application/MfciPolicy/UefiVariableSupport/UefiVariablesSupportLib.py

```diff
@@ -0,0 +1,169 @@
+# @file
+#
+# Python lib to support Reading and writing UEFI variables from windows
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+
+from ctypes import (
+    windll,
+    c_wchar_p,
+    c_void_p,
+    c_int,
+    c_char,
+    create_string_buffer,
+    WinError,
+)
+import logging
+from win32 import win32api
+from win32 import win32process
+from win32 import win32security
+
+kernel32 = windll.kernel32
+EFI_VAR_MAX_BUFFER_SIZE = 1024 * 1024
+
+
+class UefiVariable(object):
+    def __init__(self):
+        # enable required SeSystemEnvironmentPrivilege privilege
+        privilege = win32security.LookupPrivilegeValue(
+            None, "SeSystemEnvironmentPrivilege"
+        )
+        token = win32security.OpenProcessToken(
+            win32process.GetCurrentProcess(),
+            win32security.TOKEN_READ | win32security.TOKEN_ADJUST_PRIVILEGES,
+        )
+        win32security.AdjustTokenPrivileges(
+            token, False, [(privilege, win32security.SE_PRIVILEGE_ENABLED)]
+        )
+        win32api.CloseHandle(token)
+
+        # import firmware variable API
+        try:
+            self._GetFirmwareEnvironmentVariable = (
+                kernel32.GetFirmwareEnvironmentVariableW
+            )
+            self._GetFirmwareEnvironmentVariable.restype = c_int
+            self._GetFirmwareEnvironmentVariable.argtypes = [
+                c_wchar_p,
+                c_wchar_p,
+                c_void_p,
+                c_int,
+            ]
+            self._SetFirmwareEnvironmentVariable = (
+                kernel32.SetFirmwareEnvironmentVariableW
+            )
+            self._SetFirmwareEnvironmentVariable.restype = c_int
+            self._SetFirmwareEnvironmentVariable.argtypes = [
+                c_wchar_p,
+                c_wchar_p,
+                c_void_p,
+                c_int,
+            ]
+            self._SetFirmwareEnvironmentVariableEx = (
+                kernel32.SetFirmwareEnvironmentVariableExW
+            )
+            self._SetFirmwareEnvironmentVariableEx.restype = c_int
+            self._SetFirmwareEnvironmentVariableEx.argtypes = [
+                c_wchar_p,
+                c_wchar_p,
+                c_void_p,
+                c_int,
+                c_int,
+            ]
+        except:

```

Comment:
no bare excepts - You can do a blanket `except Exception`

https://docs.astral.sh/ruff/rules/bare-except/